index.js

```diff
@@ -184,7 +185,27 @@ function resolveWebpackPath(partial, filename, directory, webpackConfig) {
 
   try {
     var loadedConfig = require(webpackConfig);
-    var resolver = webpackResolve.create.sync(loadedConfig.resolve || {});
+    var resolveConfig = objectAssign({}, loadedConfig.resolve);
```

Comment:
Non-blocker: Since filing-cabinet is in the hot code path for dependency-tree, a for-in loop might be preferable. Probably negligible, but something to keep in mind for code in precinct, cabinet, and its dependencies.